### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by someshsingh22**
Typo Try Except Removed, excess code removed, random.sample() in place of random.randint

Note : The coverage error is still there from issue https://github.com/SforAiDl/decepticonlp/issues/28

### CODE REVIEW

File: decepticonlp/transforms/perturbations.py [hunk 1]
```diff
@@ -251,23 +240,19 @@ def apply(self, word: str, **kwargs):
...
-        # reinsert capitalization
-        for i in range(len(word)):
-            if capitalization[i]:
-                word[i] = word[i].upper()
+            # Check if in position and given keys
+            if i in positions_to_shift and c in keys_in_proximity:
+                word[i] = random.choice(keys_in_proximity[c])
+                if cap:
+                    # convert to upper if in upper
+                    word[i] = word[i].upper()
```
Comment (rajaswa) [inline decepticonlp/transforms/perturbations.py:252]: Add test case to cover this also.
That should resolve the coverage failure?

Comment (someshsingh22) [reply decepticonlp/transforms/perturbations.py:252]: The test case is already there, does code coverage count the number of test cases for logic ?

Comment (someshsingh22) [reply decepticonlp/transforms/perturbations.py:252]: A simpler resolve would be to extend the dictionary to capital letters.

Pros - Defining non alphabetical typos as well, the current implementation and this PR to assume that typos consider letters only, however, I can also put a number in between. So we can define different typos for capital letters

Cons - Will clutter the code.

Should I save this dictionary as a son rather ? This way in future if we wish to add any changes we don't have to mess with the code

**Comment by someshsingh22** [comment]
This Pull request will be completed in https://github.com/SforAiDl/decepticonlp/pull/52

**Pull request closed without merging**